bundle/manifests/nvidia-maintenance-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,377 @@
+              "app.kubernetes.io/name": "nodemaintenance",
+              "app.kubernetes.io/part-of": "maintenance-operator"
+            },
+            "name": "nodemaintenance-sample"
+          },
+          "spec": {
+            "cordon": true,
+            "drainSpec": {
+              "deleteEmptyDir": true,
+              "force": true,
+              "podEvictionFilters": [
+                {
+                  "byResourceNameRegex": "nvidia.com/gpu-*"
+                },
+                {
+                  "byResourceNameRegex": "nvidia.com/rdma*"
+                }
+              ],
+              "podSelector": "app=rdma-workload",
+              "timeoutSeconds": 0
+            },
+            "nodeName": "worker-node-01",
+            "requestorID": "some-cluster-component.vendor.com",
+            "waitForPodCompletion": {
+              "podSelector": "app=rdma-workload",
+              "timeoutSeconds": 0
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    createdAt: "2024-12-24T10:44:56Z"
+    description: Node maintenance in K8s cluster in a coordinated manner
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    operatorframework.io/suggested-namespace: nvidia-maintenance-operator
+    operators.operatorframework.io/builder: operator-sdk-v1.37.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v4
+    provider: NVIDIA
+    repository: https://github.com/Mellanox/maintenance-operator/
+    support: NVIDIA
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.arm64: supported
+  name: nvidia-maintenance-operator.v0.1.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: MaintenanceOperatorConfig is the Schema for the maintenanceoperatorconfigs
+        API
+      displayName: Maintenance Operator Config
+      kind: MaintenanceOperatorConfig
+      name: maintenanceoperatorconfigs.maintenance.nvidia.com
+      version: v1alpha1
+    - description: NodeMaintenance is the Schema for the nodemaintenances API
+      displayName: Node Maintenance
+      kind: NodeMaintenance
+      name: nodemaintenances.maintenance.nvidia.com
+      version: v1alpha1
+  description: Coordinates And Performs Common Node Preparations For Maintenance
+  displayName: NVIDIA Maintenance Operator
+  icon:
+  - base64data: ""
```

Comment:
maybe it's a good idea to put NVIDIA logo or NVIDIA infrastructure logo here. Let's clarify with Alex P